xray_test.go

```diff
@@ -106,7 +106,24 @@ func TestXrayAuditNuget(t *testing.T) {
 	assert.NoError(t, exec.Command("nuget", "restore").Run())
 
 	output := xrayCli.RunCliCmdWithOutput(t, "audit", "--format=json")
-	verifyScanResults(t, output, 0, 1, 0)
+	verifyScanResults(t, output, 0, 2, 0)
+}
+
+// Tests NuGet audit by providing a multi-project NuGet project and asserts any error.
+func TestXrayAuditNugetMultiProject(t *testing.T) {
```

Comment:
It looks like this function has some common code with the ```TestXrayAuditNuget``` function above, so let's see if we can take the common code out and reuse it.